### PR TITLE
Add icETH and ACX token

### PR DESCRIPTION
**Pull request by fleupold**
As discussed on slack. Not sure if we need to do something to make sure the images are long lived (e.g. add them to IPFS ourselves).

### CODE REVIEW

File: src/public/CowSwap.json [hunk 1]
```diff
@@ -436,6 +436,22 @@
...
       "chainId": 1,
       "logoURI": "https://gateway.pinata.cloud/ipfs/QmSxuQpx3UadQY4B2i5AQAaU97JZqkhcm9dCZXXNsNziYR"
+    },
+    {
+      "symbol": "icETH",
+      "name": "Interest Compounding ETH Index",
+      "address": "0x7C07F7aBe10CE8e33DC6C5aD68FE033085256A84",
+      "decimals": 18,
+      "chainId": 1,
+      "logoURI": "https://assets.website-files.com/60d237b6ea65be721ff4bbfa/62388a82c46d9a1dfb3dc506_icETH-token-logo.png"
```
Comment (nlordell) [inline src/public/CowSwap.json:446]: nit: All of our logos are uploaded to IPFS (which ensures availability even if they are no longer accessible at the specified URL). I think we should do the same for this here.

File: src/public/CowSwap.json [hunk 2]
```diff
@@ -436,6 +436,22 @@
...
+    },
+    {
+      "symbol": "ACX",
+      "name": "Across Protocol Token",
+      "address": "0x44108f0223A3C3028F5Fe7AEC7f9bb2E66beF82F",
+      "decimals": 18,
+      "chainId": 1,
+      "logoURI": "https://raw.githubusercontent.com/balancer-labs/assets/master/assets/0x44108f0223a3c3028f5fe7aec7f9bb2e66bef82f.png"
```
Comment (nlordell) [inline src/public/CowSwap.json:454]: Idem.

File: src/public/CowSwap.json (hunk 1 above)
Comment (fleupold) [reply src/public/CowSwap.json:446]: I thought so too (mentioned in the PR description) however it's not true that all other token images are hosted on IPFS (a lot of them are hosted via the 1Inch list). We could also host them in this repo. 

I can take the time to convert those and also write a proper readme if we agree on a process.

Comment (nlordell) [reply src/public/CowSwap.json:446]: Ah, sorry - I just looked at the ones immediately above the added tokens.

**Review by nlordell — APPROVED** [review]
LGTM.

I would update logos of the added tokens to our Pinata (or even GitHub - the important thing is we should control the file hosting).

The downside of this of course is that we lose out on "automatic logo updates" that may be pushed to other sources (such as the 1Inch list).

**Pull request merged**